doc/usermanual/darkroom/modules/tone/tonecurve.xml

```diff
@@ -68,11 +67,74 @@
       below) at the selected spot or area are shown on top left of the widget.
     </para>
 
+    <sect5 status="draft">
+      <title>color spaces</title>
+      <para>
+        Depending on the desired intent, you can apply the tone curve in 3 different color spaces:
+        <itemizedlist>
+          <listitem>Lab, linked channels,</listitem>
```

Comment:
Oops, xmllint still complaining. The content of each listitem needs to be enclosed in `<para></para>`.

Reply:
it should be good now